tests/modeladmin/tests.py

```diff
@@ -615,6 +617,49 @@ def test_log_actions(self):
         self.assertEqual(ma.log_change(mock_request, self.band, 'changed'), LogEntry.objects.latest('id'))
         self.assertEqual(ma.log_change(mock_request, self.band, 'deleted'), LogEntry.objects.latest('id'))
 
+    @mock.patch('django.contrib.admin.options.get_deleted_objects')
```

Comment:
It was because of the comments in the ticket on the originally submitted patch - that it needs to test that the `ModelAdmin.get_deleted_objects()` method is actually used.  The behaviour of the `get_deleted_objects()` function is tested elsewhere, so it didn't seem necessary to test it again here. I just wanted to test that calling the new method actually called that function.  The other test checks the opposite - that overriding the ModelAdmin method doesn't call it.